br/pkg/utils/math_test.go

```diff
@@ -5,21 +5,22 @@ package utils
 import (
 	"testing"
 
+	tidbmath "github.com/pingcap/tidb/util/math"
 	"github.com/stretchr/testify/require"
 )
 
 func TestMinInt(t *testing.T) {
-	require.Equal(t, 1, MinInt(1, 2))
-	require.Equal(t, 1, MinInt(2, 1))
-	require.Equal(t, 1, MinInt(4, 2, 1, 3))
-	require.Equal(t, 1, MinInt(1, 1))
+	require.Equal(t, 1, tidbmath.Min(1, 2))
```

Comment:
if you implement this in util/math, please move all related tests to util/math_test.go, not in br folder now.

Reply:
Done